restful-users-exercise/src/main/java/com/teksystems/users/restfulusersexercise/Controller/UserController.java

```diff
@@ -0,0 +1,73 @@
+package com.teksystems.users.restfulusersexercise.Controller;
+
+import com.teksystems.users.restfulusersexercise.Model.Request.UserRequest;
+import com.teksystems.users.restfulusersexercise.Model.Response.UserResponse;
+import com.teksystems.users.restfulusersexercise.Model.User;
+import com.teksystems.users.restfulusersexercise.Services.UserService;
+import com.teksystems.users.restfulusersexercise.Shared.Dto.UserDto;
+import org.springframework.beans.BeanUtils;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("users")
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping
+    public List<User> getUsers(@RequestParam(value = "page", defaultValue = "1")int page,
+                               @RequestParam(value = "limit", defaultValue = "2")int limit){
+        List<User> returnValue = userService.getUsers(page,limit);
+        return returnValue;
+    }
+
+    @GetMapping(path = "/email/{email}")
```

Comment:
redundant, use `{email}`. Also try not to expose private data like email address in a URI. Get it from the request body instead